native/cocos/network/HttpClient-apple.mm

```diff
@@ -149,7 +149,7 @@ static int processTask(HttpClient *client, HttpRequest *request, NSString *reque
             unsigned long i = header.find(':', 0);
             unsigned long length = header.size();
             ccstd::string field = header.substr(0, i);
-            ccstd::string value = header.substr(i + 1, length - i);
+            ccstd::string value = header.substr(i + 2);
```

Comment:
We should trim the value field since it's not ensured that the value always starts with a white space.

Reply:
done.